docs/concepts/philosophy.rst

```diff
@@ -1,2 +1,40 @@
 Philosophy Behind j5
-====================
\ No newline at end of file
+====================
+
+Some Background
+---------------
+
+Student Robotics is a charity that was originally founded by a group of students at the University of Southampton with
+the goal of bringing the excitement of engineering and the challenge of coding to young people through robotics. This
+has involved running an annual robotics competition almost every year since 2008, where groups of sixth form students
+are given some robotics kit, time and mentoring to develop a competitive robot for a unique challenge. In order to
+reduce the barrier to entry for the competition, it is essential that a knowledge of low level programming and
+hardware is not required by the students. Thus, a Python API is usually supplied alongside hardware that is developed
+in order to make things easier.
```

Comment:
This implies that Python is somehow easy to understand or reason about. As Python is an imperative language with side effects, this is demonstrably false.

Reply:
I'm disregarding this.